ginepro/src/balanced_channel.rs

```diff
@@ -55,8 +59,8 @@ impl LoadBalancedChannel {
     /// All the service endpoints of a [`ServiceDefinition`] will be
     /// constructed by resolving IPs for [`ServiceDefinition::hostname`], and
     /// using the port number [`ServiceDefinition::port`].
-    pub async fn builder<H: Into<ServiceDefinition>>(
-        service_definition: H,
+    pub async fn builder(
+        service_definition: ServiceDefinition,
```

Comment:
Why not a `H: TryInto<ServiceDefinition>` bound?

Reply:
Good idea!